omnibridge/wrappers/api_based_wrappers/gpt_wrapper.py

```diff
@@ -1,55 +1,65 @@
-from typing import Literal, TypedDict, Dict
+from typing import Literal, TypedDict, Dict, Any
 import requests
 import json
 
+from .base_api_wrapper import RestAPIWrapper
 from ..models_configurations.chatgpt_config import GPTConfiguration, chatGptModel
 
-
 COMPLETIONS_API_URL = "https://api.openai.com/v1/chat/completions"
 
+
 class GPTWrapperException(Exception):
     pass
 
+
 class CompletionsRequestBody(TypedDict):
     model: chatGptModel
     messages: list[dict[Literal["role", "content"], str]]
 
 
-class GPTWrapper:
-    def __init__(self, prompt: str, configuration: GPTConfiguration) -> None:
-        self.prompt = prompt
+class GPTWrapper(RestAPIWrapper):
+
+    def __init__(self, configuration: GPTConfiguration) -> None:
+        super().__init__(configuration)
         self.config = configuration
+        self.api_url = COMPLETIONS_API_URL
+
+    def _get_api_url(self) -> str:
+        pass
```

Comment:
Do we have a use case for this function? I see we already define the attribute `self.api_url` in `__init__`